app/src/main/java/edu/uga/eu_soccer_explorer/DBEngine.java

```diff
@@ -13,8 +13,15 @@
 public class DBEngine {
 
     // JDBC driver name
-    static final String JDBC_DRIVER = "com.mysql.jdbc.Driver";
+    //static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
```

Comment:
```suggestion
    //static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
```I guess we should clean all these unnecessary comments.